maxim/dataset/dataset.py

```
@@ -109,3 +115,102 @@ def validate_data_structure(
                     "platformDataStructureKeys": list(against_data_structure_keys),
                 },
             )
+
+
+def add_entries(api: "MaximAPI", dataset_id: str, dataset_entries: Union[List[DatasetEntry], List[Dict[str, Any]]]) -> dict[str, Any]:
```

Ruff (0.12.2)

`add_entries` is too complex (14 > 10)

(C901)